crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -673,6 +674,21 @@ impl CheckpointStore {
         Ok(())
     }
 
+    pub fn get_epoch_state_commitments(
```

Comment:
This function is a bit special and only make sense to be called at the end of epoch.
Probably worth making it clear in the naming so it's clear, like get_state_commitments_at_epoch_end or something